CHANGES.md

```diff
@@ -15,6 +15,9 @@ ones in. -->
 
 ### Enhancements
 
+[#5537](https://github.com/cylc/cylc-flow/pull/5537) - Allow parameters
+in the form `<foo>word<bar>` which were not previously fully supported.
```

Comment:
I think this would be a bit clearer
```suggestion
in the form `task<foo><bar>` which were not previously fully supported.
```

Reply:
`task<foo><bar>` illustrates the new syntax, but it doesn't illustrate that the new syntax allows one to have parameters in arbitrary places in the text.

Reply:
```suggestion
in family names to be split, e.g. `<foo>FAM<bar>`.
```